test/consts.test.ts

```diff
@@ -49,4 +49,20 @@ describe('consts', () => {
             }
         });
     });
+
+    describe('ACTOR_ENV_VARS', () => {
+        it('every value begins with "ACTOR_"', () => {
+            Object.values(ACTOR_ENV_VARS).forEach((v) => {
+                expect(v.startsWith('ACTOR_')).toBe(true);
```

Comment:
I think we can be stricter with the test
```suggestion
        it('every value is "ACTOR_" + key', () => {
            Object.entries(ACTOR_ENV_VARS).forEach(([k, v]) => {
                expect(v).toBe(`ACTOR_${k}`);
```

Reply:
smart, nice